### PR TITLE
CC-535 Apikey Docs

### PR DESCRIPTION
**Pull request by goyal1092**

### CODE REVIEW

**Comment by goyal1092** [comment]
@Tailored28 please review :)

File: Api-docs.md [hunk 1]
```diff
@@ -0,0 +1,1949 @@
+# Culture Counts api endpoints
+
+Culture counts have multiple api endpoints for fetcing, creating & editing data.
```
Comment (chr15m) [inline Api-docs.md]: "fetching"

File: Api-docs.md [hunk 2]
```diff
@@ -0,0 +1,1949 @@
+# Culture Counts api endpoints
```
Comment (chr15m) [inline Api-docs.md]: Throughout document "API" should be capitalized I think.

File: Api-docs.md [hunk 3]
```diff
@@ -0,0 +1,1949 @@
...
+	 		- Editor : Have some permissions to edit survey and can only edit viewer type shares
+	 		- Viewer : Can only view survey and survey details
+
+	 	- Sharer Types:
+	 		- user : share tareget evaluation/survey/customreport with user
+	 		- organisation : share tareget evaluation/survey/customreport with orgaisation
+
+	 	- Target Types:
+	 		- evaluation : user/organisation will get access to all survey's of shared evaluation
+	 		- suvrey : user/organisation will get access to a specific survey data
```
Comment (chr15m) [inline Api-docs.md]: suvrey = "survey"

File: Api-docs.md [hunk 4]
```diff
@@ -0,0 +1,1949 @@
...
+	 		- customreport : user/organisation will get access to a specific custom minisite report data
+
+	 	- Tips:
+	 		- Targret and sharer are required when creating a share.
```
Comment (chr15m) [inline Api-docs.md]: Targret = "Target"

Might be good to run this doc through spell check.

File: README.md [hunk 5]
```diff
@@ -203,24 +203,35 @@ User can fetch data using 3 type of queries:
...
+  * `authtoken`:  you can `authtoken` from `/api/auth/satatus/` or from login response.
+
 ####  PUT Request
 
+1) URL : URL of api request like `https://gaurav.new-dev.culturecounts.cc/api/user/<ID>/`
+2) ID : Id of the object you want to edit
+3) Payload : Data you want to change like name of evaluation, survey etc.
+3) Headers : Required headers to do post request
+  * `Content-Type` : `application/json`
+  * `authtoken`:  you can `authtoken` from `/api/auth/satatus/` or from login response.
```
Comment (chr15m) [inline README.md]: `/api/auth/status/`

File: README.md [hunk 6]
```diff
@@ -203,24 +203,35 @@ User can fetch data using 3 type of queries:
...
+
+  Without these headers you won't be able to PUT requests to the culture counts api
+
 ####  DELETE Request
 
+1) URL : URL of api request like `https://gaurav.new-dev.culturecounts.cc/api/user/<ID>/`
+2) ID : Id of the object you want to edit
+3) Headers : Required headers to do post request
+  * `Content-Type` : `application/json`
+  * `authtoken`:  you can `authtoken` from `/api/auth/satatus/` or from login response.
```
Comment (chr15m) [inline README.md]: `/api/auth/status/`

**Review by chr15m — CHANGES_REQUESTED** [review]
Some small changes, spelling etc.

File: Api-docs.md (hunk 4 above)
Comment (goyal1092) [reply Api-docs.md]: 👍 
Thanks @chr15m

**Comment by goyal1092** [comment]
@chr15m Updated PR. 
Thanks for the review

**Review by chr15m — APPROVED** [review]
LGTM

**Pull request merged**